src/jdk.incubator.foreign/share/classes/jdk/internal/foreign/MemoryAddressImpl.java

```diff
@@ -94,9 +97,11 @@ public String toString() {
     }
 
     @Override
+    @CallerSensitive
+    @NativeAccess
```

Comment:
I don't think we need @NativeAccess here - this is not an API method

Reply:
removed. Added @NativeAccess for an API method (missing earlier)